src/source_drivers/modbus/services/rtu_polling.py

```diff
@@ -55,9 +55,7 @@ def __poll(self):
         results = self.__get_all_points()
         # TODO: separate thread for each network
         for network, device, point in results:
-            if all(v is not None for v in results):
-                time.sleep(RtuPolling._polling_period)
-                poll_point(self, network, device, point, ModbusType.RTU)
+            poll_point(self, network, device, point, ModbusType.RTU)
```

Comment:
May be we get issue like previously we had so we were checking all these should be not `None`. But it's fine to leave this as is for now.

Reply:
I do try/excepts in `poll.py` instead.
tested by trying to read and update to deleted points after already fetched in loop and catches fine